src/mirror.c

```diff
@@ -74,7 +74,7 @@ static int write_to_path(char *content, char *path)
 	dir = dirname(tmp);
 	/* attempt to  make the directory
 	 * ignore EEXIST errors */
-	int ret = mkdir(dir, S_IRWXU);
+	int ret = mkdir(dir, S_IRWXU | S_IRGRP | S_IXGRP | S_IROTH | S_IXOTH);
```

Comment:
OK, I am very confused - nothing unusual there.

We have on line 77 something to make a directory, and then on lines 79 and 80 we make the same directory.

Reply:
Hmmm... tunnel vision; I didn't even notice that.

Reply:
That'd be my fault. Looks like I switched methods and forgot to remove the old way.